src/main/java/finance/Finance.java

```diff
@@ -1,47 +1,51 @@
+
 package finance;
 
-import helpers.*;
+import api.API;
+import com.toedter.calendar.JDateChooser;
 import org.jfree.chart.ChartPanel;
 
 import javax.swing.*;
 import java.awt.*;
 import java.awt.event.ActionEvent;
-import java.sql.*;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
 import java.text.DecimalFormat;
 import java.text.NumberFormat;
 import java.time.LocalDate;
+import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
 import java.util.*;
 import java.util.concurrent.TimeUnit;
-import java.util.Date; // Tells code to use java.util.Date rather than java.sql.Date
```

Comment:
I don't know if this is necessary, but it might prevent accidental errors if we put it back